.github/workflows/adhoc-prerelease.yml

```diff
@@ -1,13 +1,16 @@
-name: Pages
+name: Adhoc Prerelease
 
 on:
-  push:
-    branches:
-      - pages
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag"
+        required: true
```

Comment:
I guess that `required: true` allows the empty string?
Otherwise the `if:` further down would have no effect...